cpp/src/Slice/Parser.cpp

```diff
@@ -2064,6 +2064,25 @@ Slice::Module::createStruct(const string& name, NodeType nt)
     return p;
 }
 
+TypeAliasPtr
+Slice::Module::createTypeAlias(const string& name, const TypePtr& type, const StringList& metadata)
+{
```

Comment:
Should we assert that type is not an alias itselft?

Reply:
Yes, it couldn't hurt. I've added an assert here.